test/tools/libtesteth/TestHelper.cpp

```diff
@@ -460,6 +460,34 @@ void userDefinedTest(std::function<json_spirit::mValue(json_spirit::mValue const
 	}
 }
 
+void executeTests2(test::testType _type, const std::string& _testFolder)
```

Comment:
execute test files of a given type in a given folder. 
we have boost_test_case(stInitCode)  for example where stInitCode is a folder name with tests for that test suite.